README.md

```diff
@@ -97,6 +97,9 @@ All code contributions - including those of people having commit access - must g
 
 We truly ❤️ pull requests! If you wish to help, you can learn more about how you can contribute to this project in the [contribution guide](CONTRIBUTING.md).
 
+## License
+This repository is available under the [BSD 3-Clause License](https://github.com/appwrite/appwrite/blob/master/LICENSE).
```

Comment:
Please leave one blank line below a heading, to avoid MD022 (`blanks-around-headers Headings should be surrounded by blank lines`)

Reply:
@srivastavaayu, please use relative paths in URL's wherever possible.